xil_carma_cloud/docker-compose.yml

```diff
@@ -409,7 +409,7 @@ services:
       - ./mysql:/docker-entrypoint-initdb.d
 
   php:
-    image: usdotfhwaops/php:7.6.0
+    image: ${OPS_DOCKER_ORG:-usdotfhwaops}/php:${OPS_DOCKER_TAG:-develop}
```

Comment:
Do we have OPS_DOCKER_ORG in .env?

Reply:
No because we use the default value. We could define/override it in the .env file.